Prac_08/silver_service_taxi.py

```diff
@@ -0,0 +1,23 @@
+"""
+CP1404 Practical 8
+Silver taxi class
+Henry Vooren
+"""
+from Prac_08.taxi import Taxi
+
+
+class SilverTaxi(Taxi):
+
+    def __init__(self, name, fuel, fanciness_rating):
+        super().__init__(name, fuel)
+        self.current_fare_distance = 0
+        self.fanciness_rating = fanciness_rating
+        self.flagfall = 4.50

```

Comment:
This should be a shared class variable, not an instance variable.
Incorrect "reliability" field name.